src/docs/development/data-and-backend/json.md

```diff
@@ -184,7 +188,7 @@ itself. With this new approach, you can decode a user easily.
 
 <!-- skip -->
 ```dart
-Map userMap = json.decode(json);
+Map userMap = json.decode(string_with_json);
```

Comment:
In the [Library Tour](https://www.dartlang.org/guides/libraries/library-tour#dartconvert---decoding-and-encoding-json-utf-8-and-more), we use the variable name `jsonString`. Could you change it to that name?